packages/mobile/src/common/types/queries/products-query.ts

```diff
@@ -0,0 +1,9 @@
+import { ProductType } from '@vse-bude/shared';
+
+type ProductQuery = {
```

Comment:
```suggestion
type ProductRequestDto = {
```

End let's put it into product's folder